app/trainings/traininglist.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="trainings-wrapper">
+	<div class="training-wrapper" *ngFor="let trainingData of trainings">
```

Comment:
let is a type of variable declaration and these types are hidden while not in use and thus performance improves.